scripts/tendermint/start.sh

```diff
@@ -26,7 +26,7 @@ docker run --rm \
 
 # make sure we allow cors origins, only possible by modifying the config file
 # https://github.com/tendermint/tendermint/issues/3216
-sed -ie 's/cors_allowed_origins.*$/cors_allowed_origins = ["*"]/' "${TMP_DIR}/config/config.toml"
+sed -ie 's/^cors_allowed_origins.*$/cors_allowed_origins = ["*"]/' "${TMP_DIR}/config/config.toml"
```

Comment:
Might want to use something like this for future-proofing?
```
sed -ie 's/^cors_allowed_origins = .*$/cors_allowed_origins = ["*"]/' "${TMP_DIR}/config/config.toml"
```

Reply:
Indeed. The space and `=` works fine for the `timeout_*` variables in `scripts/simapp/generate_template.sh`